liota/core/package_manager.py

```diff
@@ -694,8 +722,12 @@ def _package_load(self, file_name, checksum=None, autoload_flag=False,
 
         klass = getattr(module_loaded, "PackageClass")
         package_record = PackageRecord(file_name)
-        if not package_record.set_instance(klass()):
-            log.error("Unexpected failure initializing package")
+        try:  # Create instance
```

Comment:
This is a fix for "liotad becomes unresponsive (DoS) when an erroneous package is load".
https://jira-euc.eng.vmware.com/jira/browse/IOT-2111

Inside the erroneous package, there is only run(), while no clean_up(). Then, the PackageClass instance creation will fail since clean_up() only has abstract class. An system exception will be raised when we try to run .set_instance(), so without "try..catch", nobody deals with the .set_instance() exception which will cause no further command could be input and processed.  log.error("Unexpected failure initializing package") cannot take effect.

In a word, we have to explicitly use "try..except" for .set_instance().